cfssl.tf

```diff
@@ -1,6 +1,6 @@
 // IAM instance role
 resource "aws_iam_role" "cfssl" {
-  name                 = "${var.cluster_name}_cfssl"
+  name                 = "${var.iam_prefix}${var.cluster_name}-cfssl"
```

Comment:
Don't you want a dash between `${var.iam_prefix}` and `${var.cluster_name}`?

Reply:
No, because default prefix is empty. If module user wants dashes, he(us xD) can provide them